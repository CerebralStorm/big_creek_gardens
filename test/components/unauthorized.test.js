import React from 'react'
import Unauthorized from '../../app/javascript/components/unauthorized'

test('render a Unauthorized', () => {
  const wrapper = shallow(<Unauthorized />);
  expect(wrapper).toMatchSnapshot();
});