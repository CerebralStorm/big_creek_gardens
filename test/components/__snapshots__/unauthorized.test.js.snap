// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a Unauthorized 1`] = `
<div
  className="container section text-center"
>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <div
        className="error-template"
      >
        <h1>
          Oops!
        </h1>
        <h2>
          Unauthorized
        </h2>
        <div
          className="error-details"
        >
          Sorry, are not authorized to perform this action!
        </div>
        <div
          className="error-actions"
        >
          <a
            className="btn btn-primary btn-lg m-2"
            href="/"
          >
            <span
              className="glyphicon glyphicon-home"
            />
            Take Me Home 
          </a>
          <a
            className="btn btn-primary btn-lg m-2"
            href="/contact"
          >
            <span
              className="glyphicon glyphicon-envelope"
            />
            Contact Support
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
